modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,6 +90,10 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Export SAML Metadata
+
+If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. You can then upload this file to your identity provider to configure Anypoint Platform.
```

Comment:
"an XML file of your SAML configuration settings": I would put the emphasis on the settings, like this: "to download your SAML configuration settings in an XML file". Otherwise, I wait until the end of the sentence to read why I'd want to download this file.

Reply:
```suggestion
If your identity provider supports uploading service-provider metadata, you can click the *Anypoint service provider metadata* link to download your SAML configuration settings in an XML file. You can then upload this file to your identity provider to configure Anypoint Platform.
```